index.html

```diff
@@ -1097,7 +1069,7 @@ <h2>
             "PERFORMANCE-TIMELINE-2#dfn-queue-a-performanceentry">Queue</a>
             |entry|.
           </li>
-          <li>Add |entry| to |global|'s <a data-cite=
+          <li>[=Add a PerformanceResourceTiming entry|Add=] |entry| to |global|'s <a data-cite=
           "PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance
```

Comment:
ditto exported?

Reply:
Filed https://github.com/w3c/performance-timeline/issues/196. Let's deal with it in a followup, as this is not the only place citing it.